.travis.yml

```diff
@@ -13,6 +13,7 @@ install:
 jobs:  
   include:
     - stage: Test
+      if: branch != master
```

Comment:
Why have you changed the tests to not run on master? That feels like a risk, allowing your deployment if the tests would have failed!